packages/block-library/src/latest-posts/edit.js

```diff
@@ -163,13 +183,21 @@ class LatestPostsEdit extends Component {
 				</BlockControls>
 				<ul
 					className={ classnames( this.props.className, {
+						'wp-block-latest-posts__list': true,
 						'is-grid': postLayout === 'grid',
 						'has-dates': displayPostDate,
 						[ `columns-${ columns }` ]: postLayout === 'grid',
 					} ) }
 				>
 					{ displayPosts.map( ( post, i ) => {
 						const titleTrimmed = post.title.rendered.trim();
+						let excerpt = post.excerpt.rendered;
+						if ( post.excerpt.raw === '' ) {
+							excerpt = post.content.raw;
+						}
+						const temp = document.createElement( 'div' );
```

Comment:
`temp` is not a good name, let's change it to excerptElement. It then reads nice:

```
const excerptElement = document.createElement( 'div' );
excerptElement.innerHTML = excerpt;
excerpt = excerptElement.textContent || excerptElement.innerText || '';
```